network-api/networkapi/management/commands/flush_models.py

```diff
@@ -21,21 +20,6 @@ def handle(self, *args, **options):
 
         self.stdout.write('Flushing models from the database...')
 
-        self.stdout.write('Dropping Homepage objects..')
-        Homepage.objects.all().delete()
-
-        self.stdout.write('Dropping LandingPage objects...')
-        LandingPage.objects.all().delete()
-
-        self.stdout.write('Dropping Signup objects...')
-        Signup.objects.all().delete()
-
-        self.stdout.write('Dropping CampaignPage objects...')
-        Campaign.objects.all().delete()
-
-        self.stdout.write('Dropping Highlight objects...')
```

Comment:
don't we still need `Highlight` object cleanup? We create them in `load_fake_data`

Reply:
yup, good catch